src/app/pages/home/home.component.ts

```diff
@@ -1,12 +1,18 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { MoviesService } from '../../shared/movies.service';
+import { first } from 'rxjs';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss'],
 })
 
-export class HomeComponent {
+export class HomeComponent implements OnInit {
+  constructor(public readonly moviesService: MoviesService) {
+  }
 
+  ngOnInit() {
+    this.moviesService.movies$.pipe(first());
```

Comment:
You don't need first() if you are using async pipe.
At the same time, I would recommend rolling back the movies variable. It helps keep service and component more independent rather than using variable inside service